.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Docker Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  release_cass_operator:
+    name: Release Cass Operator
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login Skopeo DockerHub
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | skopeo login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin docker.io
```

Comment:
where does `skopeo` come from? it's already part of `ubuntu-latest`?

Reply:
Skopeo is installed by default, like aws utils. So yes? Not sure what magic gives us those, but it is there, version 1.1.1.